src/styles/blocks/card.scss

```
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+
+  padding: 0 16px 16px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+
+    margin: 32px auto 40px;
```

Has to be 0 auto 40. 
That distance you set as margin (32px) should use as a padding for .card

```suggestion
    margin: 0 auto 40px;
```